chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -162,11 +197,11 @@
           os = node[:provisioner][:default_os]
         end
 
-        append << node[:provisioner][:available_oses][os][:append_line]
+        append << node[:provisioner][:available_oses][os][arch][:append_line]
 
         node_cfg_dir="#{tftproot}/nodes/#{mnode[:fqdn]}"
         node_url="#{provisioner_web}/nodes/#{mnode[:fqdn]}"
-        os_url="#{provisioner_web}/#{os}"
+        os_url="#{provisioner_web}/#{os}/#{arch}"
```

Comment:
Style/SpaceAroundOperators: Surrounding space missing for operator `=`. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)